Automattic/wp-calypso
# newsletter flow: Use webp images

## Proposed Changes

* For the /setup/newsletter flow, convert pngs to webp, then change css to preferentially load the webps if supported
* We noticed downloading the almost 1MB newsletter-intro-bottom took up a large amount of time on the webpagetest for the newsletter flow
* I used `cwebp -q 85 $file -o $output` to convert
* ~~I did use fallback to png if webp is not supported - not sure if this is necessary?~~
  * https://caniuse.com/webp
  * https://caniuse.com/css-featurequeries  
  


| Filename                                  | Original Size (KB) | WebP Size (KB) | Reduction |
| ---------------------------------------- | ------------------ | -------------- | --------- |
| newsletter-intro-bottom-mobile.png       | 321.9              | 27.0          | 91%       |
| newsletter-intro-bottom.png              | 951.2              | 65.2          | 93%       |
| newsletter-processing-left-desktop.png   | 389.8              | 33.0          | 92%       |
| newsletter-processing-left.png           | 150.9              | 34.1          | 77%       |
| newsletter-processing-right-desktop.png  | 628.1              | 54.6          | 91%       |
| newsletter-processing-right.png          | 232.1              | 55.2          | 76%       |

## Testing Instructions

* Visit /setup/newsletter, make a site, and check that the images look OK and the webp versions are getting loaded

![2023-04-26_17-56](https://user-images.githubusercontent.com/937354/234721325-fc55f86e-2034-4693-95a5-5cd0cd30b813.png)

![2023-04-26_17-59](https://user-images.githubusercontent.com/937354/234721349-92c5fdc8-9a87-4032-85b5-cea097226289.png)


## Pre-merge Checklist
